cdap-watchdog/src/main/java/co/cask/cdap/logging/run/LogSaverTwillRunnable.java

```diff
@@ -204,6 +205,7 @@ static Injector createGuiceInjector(CConfiguration cConf, Configuration hConf) {
       new AuditModule().getDistributedModules(),
       new AuthorizationEnforcementModule().getDistributedModules(),
       new AuthenticationContextModules().getMasterModule(),
+      new MessagingClientModule(),
```

Comment:
Totally get tricked by this as this class doesn't extends from `AbstractMasterTwillRunnable`